### PR TITLE
Version 2: Improve certificate delete dialog

### PR DESCRIPTION
**Pull request by OleksandrSPV**
Added hook useCertificateDeleteDialog

### CODE REVIEW

File: src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { useToast } from "@peculiar/react-components";
+import { useTranslation } from "react-i18next";
+import { useLockBodyScroll } from "react-use";
+import { CertificateDeleteDialog } from "src/components/certificate-delete-dialog";
```
Comment (donskov) [inline src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx:5]: Is it the correct import path? This is the first time you have used such an entry.

File: src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+export function useCertificateDeleteDialog() {
+  const { addToast } = useToast();
+  const { t } = useTranslation();
+
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [currentCertificatDelete, setCurrentCetificateDelete] = React.useState<
+    undefined | { id: string; name: string }
+  >();
+
+  const handleOpen = (id: string, name: string) => {
```
Comment (donskov) [inline src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx:16]: From the name of the arguments, it is not clear what exactly it is. Is it a certificate or provider ID? By the way, we need both IDs for deletion.

**Review by donskov — CHANGES_REQUESTED** [review]

File: src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx (hunk 1 above)
Comment (OleksandrSPV) [reply src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx]: Fixed here https://github.com/PeculiarVentures/fortify-tools/pull/151/commits/66850b1c238b1ab502616312c0f867b41bd34832

File: src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx (hunk 2 above)
Comment (OleksandrSPV) [reply src/dialogs/certificate-delete-dialog/useCertificateDeleteDialog.tsx]: Fixed here https://github.com/PeculiarVentures/fortify-tools/pull/151/commits/66850b1c238b1ab502616312c0f867b41bd34832

**Review by donskov — APPROVED** [review]

**Pull request merged**